Whiskey/Functions/Get-WhiskeyApiKey.ps1

```diff
@@ -42,7 +42,8 @@ function Get-WhiskeyApiKey
         Stop-WhiskeyTask -TaskContext $Context `
                          -Message ('API key ''{0}'' does not exist in Whiskey''s API key store. Use the `Add-WhiskeyApiKey` function to add this API key, e.g. `Add-WhiskeyApiKey -Context $context -ID ''{0}'' -Value $apikey`.' -f $ID) `
                          -PropertyName $PropertyName
+        return

```

Comment:
What's with all the extra `return` statements everywhere?

Reply:
I started playing around with an idea of running a build or pipeline with `-ErrorAction Ignore` and found that `Stop-WhiskeyTask` will not throw an exception if `$ErrorActionPreference` is `Ignore`. So all the tasks without a `return` after `Stop-WhiskeyTask` will continue to execute even after `Stop-WhiskeyTask` is called while the error action is `Ignore`.

Also the new pattern seems to be to always include `return` after `Stop-WhiskeyTask` so I decided to fix up all instances so the behavior was consistent.